lib/card.rb

```diff
@@ -14,11 +14,9 @@ class Card < ActiveRecord::Base
 
   has_many :references_from, :class_name => :Reference, :foreign_key => :referee_id
   has_many :references_to,   :class_name => :Reference, :foreign_key => :referer_id
-  has_many :acts, :order => :id
-  has_many :actions, :order => :id, :conditions=>{:draft => [nil,false]}
-  has_many :drafts, :order=>:id, :conditions=>{:draft=>true}, :class_name=> :Action
-
-  cache_attributes 'name', 'type_id' # review - still worth it in Rails 3?
+  has_many :acts, -> { order :id }
+  has_many :actions, -> { where( :draft=>[nil,false]).order :id }
+  has_many :drafts, { :class_name=> :Action }, -> { where( :draft=>true ).order :id }
```

Comment:
"Can't we just change the spec to get 3.2.something + up to latest 3.x.y ?"

If you mean, not supporting 4.x, that's a non starter.  Wagn needs to stay fairly close to the latest rails so that new rails sites can use decko-rails.  The vast majority of installations in the past 1.5 years use rails 4.   I'm much more open to support 3.2.x than to NOT supporting 4.x.

At some point, of course, we will need to drop 3.x  But so long as we support it in the initial releases, *old* rails sites can use *old* decko-rails versions, and when they want to upgrade their decko they have to upgrade their rails, too.  That seems sensible enough, no?